integration/setup/tenant_cluster.go

```diff
@@ -0,0 +1,135 @@
+// +build k8srequired
+
+package setup
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/giantswarm/aws-operator/integration/env"
+	"github.com/giantswarm/e2e-harness/pkg/release"
+	"github.com/giantswarm/e2etemplates/pkg/chartvalues"
+	"github.com/giantswarm/e2etemplates/pkg/e2etemplates"
+	"github.com/giantswarm/microerror"
+)
+
+func CreateTenantCluster(ctx context.Context, config Config) error {
+	vpcID, err := ensureHostVPC(ctx, config)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	err = installAWSConfig(ctx, config, vpcID)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	return nil
+}
+
+func DeleteTenantCluster(ctx context.Context, config Config) error {
+	err := config.Release.Delete(ctx, "apiextensions-aws-config-e2e")
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	return nil
+}
+
```

Comment:
This is now idempotent. Goal of this function is to return host VPC ID no matter what.
BTW should it have `ControlPlane` instead of `Host` in the name?

Reply:
Shouldn't the names then reflect the idempotency via the `ensure` wording?

Reply:
Create isn't fully idempotent yet. But it will be. So I'll change that.

Reply:
Done.